src/gc-alloc-profiler.h

```diff
@@ -0,0 +1,56 @@
+// This file is a part of Julia. License is MIT: https://julialang.org/license
+
+#ifndef JL_GC_ALLOC_PROFILER_H
+#define JL_GC_ALLOC_PROFILER_H
+
+#include "julia.h"
+#include "ios.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct FreeInfo {
+    size_t type_addr;
+    size_t count;
+};
+
+struct RawAllocResults {
+    void *allocs; // Alloc* (see gc-alloc-profiler.cpp)
```

Comment:
Can we declare the RawAlloc struct here, too? I think it'd be nice to, since it will be consumed from Julia? Of course it doesn't _actually_ matter, since no C/C++ files will be sharing it, but it signals some intention that the struct definitions are "public-facing".

Reply:
Done in https://github.com/vilterp/julia/pull/10